scriptworker/cot/verify.py

```diff
@@ -2091,11 +2091,12 @@ def format(self, record):
         context = Context()
         context.session = session
         context.config = dict(deepcopy(DEFAULT_CONFIG))
+        context.config['taskcluster_root_url'] = os.environ.get('TASKCLUSTER_ROOT_URL', 'https://taskcluster.net')
```

Comment:
We can do this `os.environ.get` in `scriptworker.constants.DEFAULT_CONFIG` for expediency.